app/main/routes.py

```diff
@@ -157,6 +157,11 @@ def start_page():
     return render_template("start-page.html")
 
 
+@bp.route("/sign-out", methods=["GET"])
```

Comment:
"Sign Out" seems misleading, could we change this to "signed out" as well as anywhere else we refer to sign out?